<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editStation</class>
 <widget class="QDialog" name="editStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Station Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3" colspan="3">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Location:</string>
     </property>
     <widget class="QRadioButton" name="rbClosestPoint">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add station at closest point</string>
      </property>
      <property name="text">
       <string>Closest point</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbAddPoint">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add a new point for station</string>
      </property>
      <property name="statusTip">
       <string>A new point will be created on the line.</string>
      </property>
      <property name="text">
       <string>Add new point</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbClosestEnd">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Set station at closest end point.</string>
      </property>
      <property name="text">
       <string>Closest end point</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QDateEdit" name="dateStart">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QDateEdit" name="dateEnd">
     <property name="displayFormat">
      <string>yyyy/MM/dd</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="txtGeodbLocId">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblErrorText">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>159</red>
           <green>158</green>
           <blue>158</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QPushButton" name="btnEditText">
     <property name="text">
      <string>Edit Text</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Icon:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QComboBox" name="cbIcons"/>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLineEdit" name="txtStationName"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
